### ISSUE TITLE
computedFn: wrong typings

### ISSUE THREAD

**Issue by wuifdesign** [mobxjs/mobx-utils#212]
Typehinting isn't working for me as this function has a wrong return type.

`computedFn.d.ts`

```
export declare function computedFn<T extends Function>(fn: T, keepAlive?: boolean): (...args: any[]) => any;
```

should be 

```
export declare function computedFn<T extends Function>(fn: T, keepAlive?: boolean): T;
```

if i do something like this: `computedFn((key: string) => key)` i dont get the typehint für the string as it will be typehinted as `(...args: any[]) => any`.

**Comment by mweststrate** [comment]
PR welcome

**Comment by wuifdesign** [comment]
Seams to be fixed in the new version (5.5.0). fixed by 717ad1fd9011cb10410aafb80020d59f3ababd93
